linkerd/protocol/benchmark/src/main/scala/io/buoyant/protocol/AccessLoggerBenchmark.scala

```diff
@@ -0,0 +1,33 @@
+package io.buoyant.linkerd.protocol
+
+import com.twitter.finagle.Service
+import com.twitter.finagle.http._
+import com.twitter.util.Future
+import com.twitter.util.StdBenchAnnotations
+import com.twitter.logging.NullLogger
+import org.openjdk.jmh.annotations._
+import io.buoyant.linkerd.protocol.http.AccessLogger
+
+// ./sbt 'project linkerd-protocol-benchmark' jmh:run -i 20 -prof gc .*AccessLoggerBenchmark.*
+@State(Scope.Benchmark)
+class AccessLoggerBenchmark extends StdBenchAnnotations {
+  val service = AccessLogger(NullLogger) andThen Service.mk[Request, Response] { req =>
+    val rsp = Response()
+    rsp.status = Status.PaymentRequired
+    rsp.contentType = "application/json"
+    rsp.contentLength = 304374
+    Future.value(rsp)
+  }
+
+  val req = Request()
+  req.method = Method.Head
+  req.uri = "/foo?bar=bah"
+  req.host = "monkeys"
+  req.contentType = "text/plain"
+
+  @Benchmark
+  def benchmark(): Future[Response] = {
+    service(req)
```

Comment:
I think that makes sense and have now made that change. I wouldn't expect it to make a difference (and it doesn't seem to) but there's no chance of a timing bug now.